GVFS/GVFS.FunctionalTests/Tests/GitCommands/CheckoutTests.cs

```diff
@@ -237,7 +247,7 @@ public void CheckoutBranchAfterReadingAllFilesAndVerifyContentsCorrect()
                 .WithDeepStructure(this.FileSystem, this.ControlGitRepo.RootPath, compareContent: true);
 
             // Verify modified paths contents
-            GVFSHelpers.ModifiedPathsContentsShouldEqual(this.FileSystem, this.Enlistment.DotGVFSRoot, "A .gitattributes" + Environment.NewLine);
+            GVFSHelpers.ModifiedPathsContentsShouldEqual(this.FileSystem, this.Enlistment.DotGVFSRoot, "A .gitattributes\r\n");

```

Comment:
Suggestion from Kevin: Use a named constant for "\r\n" here (and possibly in the the database code as well)